.github/dependabot.yml

```diff
@@ -11,4 +11,12 @@ updates:
     interval: "weekly"
   open-pull-requests-limit: 10
   vulnerability-alerts:
-    enabled: true
\ No newline at end of file
+    enabled: true
+  labels:
+    - "ok-to-test"
+    - "dependencies"
+  ignore:
+    # ignore major and minor versions for dependencies updates
+    # Allow patches and security update.
+    - dependency-name: k8s.io/*
+      update-types: ["version-update:semver-major", "version-update:semver-minor"]
```

Comment:
I don't think we need to update every version of k8s deps but we want the security and patches update + don't update deps past k/k.

Reply:
as i understand this skip condition means to skip all dependency libs under k8s, this wont be checking anything in k/k repo. And our requirement of not update deps past k/k means not to jump the version of any package past k/k either under k8s or otherwise.
This condition wont check k/k dependencies. Instead it will altogether skip all dependency libs under k8s.io which is not the requirement. This will do completely seperate thing.

Reply:
the update types is to control what k/k deps we want. it will only check security update